### PR TITLE
[HNC-504] - Update reference in registry-publication

### PR DESCRIPTION
**Pull request by AdityaHitachi**

### CODE REVIEW

File: .github/workflows/bootstrap-image.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ jobs:
       - name: Push to Artifactory
         id: push-to-artifactory
         run: |
-          docker login ${{ env.ARTIFACTORY }} -u ${{ secrets.LDOS_ARTIFACTORY_USER }} -p ${{ secrets.LDOS_ARTIFACTORY_API_KEY }}
+          sudo docker login ${{ env.ARTIFACTORY }} -u ${{ secrets.JFROG_JARJAR_SERVICE_USERNAME  }} -p ${{ secrets.JFROG_JARJAR_SERVICE_TOKEN  }}
```
Comment (cardosov) [inline .github/workflows/bootstrap-image.yaml:37]: Should not need `sudo` anymore

Comment (AdityaHitachi) [reply .github/workflows/bootstrap-image.yaml]: yes removed.

**Comment by cardosov** [comment]
This repo has some examples in it to smoke test its functionality ([example](https://github.com/lumada-common-services/registry-publication/blob/master/.github/artifacts.yaml#L8)). Not all references have been changed to one.hitachivantara.com

**Comment by AdityaHitachi** [comment]
> This repo has some examples in it to smoke test its functionality ([example](https://github.com/lumada-common-services/registry-publication/blob/master/.github/artifacts.yaml#L8)). Not all references have been changed to one.hitachivantara.com

updated

**Review by vganpisetty — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ Usage example:
 - name: Run publication to registry
   id: registry-publication-action
   continue-on-error: false
-  uses: lumada-common-services/registry-publication@v1
+  uses: lumada-common-services/registry-publication@stable  #The stable tag always points to the latest changes
```
Comment (cardosov) [inline README.md:9]: Don't forget to create the stable tag after merge

**Review by cardosov — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (AdityaHitachi) [reply README.md:9]: yes

**Pull request merged**